### ISSUE TITLE
Remove deprecated dependency @types/rdf-js

### ISSUE THREAD

**Issue by jcoyne** [rdfjs-base/data-model#35]
```
npm WARN deprecated @types/rdf-js@4.0.2: This is a stub types definition. rdf-js provides its own type definitions, so you do not need this installed.
```

**Comment by tpluscode** [comment]
Thank you for opening this issue. Wha a coincidence that less than an hour prior #34 was opened to fix that.

About the warning; it is harmless, generated when a package is removed from DefinitelyTyped in favour of externally managed declarations. We moved to [@rdfjs/types](https://npm.im/@rdfjs/types)) but all packages continue to work to keep backwards compatibility

**Comment by bergos** [comment]
Should be fixed #38
